static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-liveblog-world-cup.js

```diff
@@ -0,0 +1,240 @@
+            blockToInsertAutoEpicAfter
+        );
+    });
+
+const setEpicLiveblogEntryTimeData = (
+    el: Element,
+    timeData: TimeData
+): void => {
+    const epicTimeEl = el.querySelector('time');
+    const epicAbsoluteTimeEl = el.querySelector('.block-time__absolute');
+
+    if (epicTimeEl && epicAbsoluteTimeEl) {
+        const epicTimeLink = epicTimeEl.parentNode;
+        if (epicTimeLink instanceof HTMLAnchorElement) {
+            epicTimeLink.href = timeData.blockHref;
+        }
+
+        epicTimeEl.setAttribute('datetime', timeData.datetime);
+        epicTimeEl.setAttribute('title', timeData.title);
+        epicTimeEl.innerHTML = timeData.date;
+        epicAbsoluteTimeEl.innerHTML = timeData.time;
+    }
+};
+
+const setupViewTracking = (el: HTMLElement, test: EpicABTest): void => {
+    // top offset of 18 ensures view only counts when half of element is on screen
+    const inView = elementInView(el, window, {
+        top: 18,
+    });
+
+    inView.on('firstview', () => {
+        logView(test.id);
+        mediator.emit(test.viewEvent);
+    });
+};
+
+const addEpicToBlocks = (epicHtml: string, test: EpicABTest): Promise<void> =>
+    getBlocksToInsertEpicAfter().then(blocksToInsertEpicAfter => {
+        blocksToInsertEpicAfter.forEach(el => {
+            getLiveblogEntryTimeData(el).then((timeData: TimeData) => {
+                fastdom.write(() => {
+                    const $epic = $.create(epicHtml);
+                    $epic.insertAfter(el);
+                    mediator.emit(test.insertEvent);
+                    $(el).removeClass(INSERT_EPIC_AFTER_CLASS);
+                    setEpicLiveblogEntryTimeData($epic[0], timeData);
+                    setupViewTracking(el, test);
+                });
+            });
+        });
+    });
+
+export const setupEpicInLiveblog = (
+    epicHtml: string,
+    test: EpicABTest
+): void => {
+    addEpicToBlocks(epicHtml, test);
+
+    if (!isAutoUpdateHandlerBound) {
+        mediator.on('modules:autoupdate:updates', () => {
+            addEpicToBlocks(epicHtml, test);
+        });
+        isAutoUpdateHandlerBound = true;
+    }
+};
+
+export const acquisitionsEpicLiveblogWorldCup: EpicABTest = makeABTest({
+    id: 'AcquisitionsEpicLiveblogWorldCup',
+    campaignId: 'epic_liveblog_world_cup',
+    campaignSuffix: pageId.replace(/-/g, '_').replace(/\//g, '__'),
+
+    start: '2018-06-21',
+    expiry: '2019-01-24',
+
+    author: 'Jonathan Rankin',
+    description:
+        'This places the epic underneath liveblog blocks which the author has specified in Composer should have an epic against them',
+    successMeasure: 'Member acquisition and contributions',
+    idealOutcome:
+        'Our wonderful readers will support The Guardian in this time of need!',
+
+    audienceCriteria: 'All',
+    audience: 1,
+    audienceOffset: 0,
+
+    pageCheck(page) {
+        return page.contentType === 'LiveBlog' && tagsMatch();
```

Comment:
I think just `keywordExists(['World Cup 2018'])` would be cleaner, and keeps it all inside `pageCheck` for readability

`import { keywordExists } from 'lib/page';` is the import

also we can ES6 this syntax: `pageCheck: (page) => ...`